appdatabase/database.go

```diff
@@ -62,20 +67,33 @@ func doMigration(db *sql.DB) error {
 }
 
 // InitializeDB creates db file at a given path and applies migrations.
-func InitializeDB(path, password string, kdfIterationsNumber int) (*sql.DB, error) {
+func InitializeDB(path, password string, kdfIterationsNumber int, keyUID string) (*sql.DB, error) {
 	db, err := sqlite.OpenDB(path, password, kdfIterationsNumber)
 	if err != nil {
 		return nil, err
 	}
 
-	err = doMigration(db)
+	err = doMigration(db, keyUID)
 	if err != nil {
 		return nil, err
 	}
 
 	return db, nil
 }
 
+func fixMissingKeyUIDForAccounts(keyUID string) *sqlite.PostStep {
+	return &sqlite.PostStep{
+		Version: 1662365868,
+		CustomMigration: func(sqlTx *sql.Tx) error {
+			if keyUID == "" {
+				return nil
+			}
+			_, err := sqlTx.Exec(`UPDATE accounts SET key_uid = ? WHERE 1=1`, keyUID)
```

Comment:
> this migration will execute after 1662365868_add_key_uid_accounts.up.sql, so i think it's okay?

Ok, thanks, I missed that.

Reply:
@qfrank I wrote a comment at the same moment as the previous one but forgot to press the button. :) 

> We separated our app db file via keyUid before this migration, if user have multi status accounts say accountA and accountB, there should be 2 separate db files, this migration runs on each db file separately, also pls note it only runs after 1662365868_add_key_uid_accounts.up.sql so there should be no problem?

@qfrank does it mean the mobile app has a single accounts.sql db with a single entry for every new Status profile? Or new accounts table in accounts.sql db for every new Status profile?
Then it's not what the desktop app has.

Reply:
@qfrank ok, thanks for this detailed explanation, initially I thought that you were trying to add `keyUid` to the `accounts` table of `accounts.sql` db. That's why I said it may contain more than a single Status profile. But now is ok, all clear regarding your intention, thanks.

Just one more thing to confirm, since this `CustomMigration` will be applied over all rows in `accounts` table:
- are we sure that all added accounts refer to the Status profile keypair (means that, for that old app version, there was no chance for a user to add an account other than the profile key pair)?
- are we sure that in that old app a user was not able to add a watch-only account, otherwise `CustomMigration` should be updated to avoid setting `keyUid` field for watch-only accounts.